### ISSUE TITLE
Mines set area argument checks for 'yes' and 'confirm' rather than either or.

### ISSUE THREAD

**Issue by dkf-lab** [PrisonTeam/Prison#223]
Hello
Running version 3.2.10

When running /mines set area <mine> I get the following prompt:
Warning: this mine size is x. Re-submit command with either 'confirm' or 'yes' to the end of the command.

So if I run:

/mines set area <mine> wand yes
I get the same message

and

/mines set area <mine> wand confirm
I get the same message.

I can get it to work by running /mines set area <mine> wand confirm yes

Basically the code is checking for yes AND confirm, rather than just one of them.

**Comment by GABRYCA** [comment]
Got, this must be a recent issue or maybe it got even fixed within Alpha, you can find it on the git builds or even AppVeyor, we need to check if this issue persists and eventually fix it.
Thank you very much for the report, please update us if you have new info/data.

**Comment by rbluer** [comment]
Thanks for reporting that issue.  That issue "was" fixed a while ago, or at least I thought it was.

I just checked the source code and yes, you are correct in your assumptions.

It was setup as:
blockSize > 25,000 AND options does not contains "yes" **OR** options does not contain "confirm"

The OR should have been an AND.

I fixed this and it will be included in the next alpha release that will be published on our discord server in the next few days.  I'm closing this issue since the code has been corrected.

Thanks for reporting this issue!
  - Blue
